package.json

```diff
@@ -71,6 +71,7 @@
     "lint-staged": "^9.5.0",
     "prettier": "^2.0.5",
     "react": "^17.0.1",
+    "react-dom": "^17.0.1",
```

Comment:
This means we probably now also need to declare `react-dom` in our `peerDependencies`

Reply:
Agree